proto/dydxprotocol/vault/genesis.proto

```diff
@@ -3,16 +3,20 @@ package dydxprotocol.vault;
 
 import "gogoproto/gogo.proto";
 import "dydxprotocol/vault/params.proto";
+import "dydxprotocol/vault/share.proto";
 import "dydxprotocol/vault/vault.proto";
 
 option go_package = "github.com/dydxprotocol/v4-chain/protocol/x/vault/types";
 
 // GenesisState defines `x/vault`'s genesis state.
 message GenesisState {
   // The parameters of the module.
-  Params params = 1 [ (gogoproto.nullable) = false ];
+  // Deprecated since v6.x in favor of default_quoting_params.
+  Params params = 1 [ deprecated = true ];
```

Comment:
nit: Any issue in using `reserved` instead?

Reply:
hmm yeah that's probably cleaner!